File: service/SurfWebAutofac/Model/Caches/MapBountyOrStageCache.cs
namespace Model.Caches;

public class MapBountyOrStageCache
{
    /// <summary>
    ///     地图ID
    /// </summary>
    public required long Id { get; set; }

    /// <summary>
    ///     地图名称
    /// </summary>
    public required string Name { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public required string Difficulty { get; set; }

    /// <summary>
    ///     图片
    /// </summary>
    public string Img { get; set; }

    /// <summary>
    ///     阶段
    /// </summary>
    public int Stage { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/caches/MapBountyOrStageCache.java
package com.surfweb.system.caches;

import lombok.Data;
/**
 * 地图奖励关或阶段缓存
 */
@Data
public class MapBountyOrStageCache {

    /**
     * 地图ID
     */
    private long id;

    /**
     * 地图名称
     */
    private String name;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 图片
     */
    private String img;

    /**
     * 阶段
     */
    private int stage;
}
